flake.nix

```diff
@@ -8,16 +8,19 @@
     # Custom packages
     polykey-cli.url = "github:MatrixAI/Polykey-CLI";
     polykey-cli.inputs.nixpkgs.follows = "nixpkgs"; # Inheriting the nixpkgs input above; pinned
+
+    gitignore-nix.url = "github:hercules-ci/gitignore.nix";
+    gitignore-nix.inputs.nixpkgs.follows = "nixpkgs";
```

Comment:
So according to the upstream package, this package name should just be `gitignore`.

I checked in nixpkgs upstream and `gitignore` is an un-used alias.

I think you should just call this `gitignore`.

Reply:
Since you've created your own `gitignore` wrapper.

Then this package should be aliased to be `hercules-gitignore`.

Reply:
I will call the final result that we can use, `gitignore`. However, this is the gitignore provided by hercules-ci. Following https://github.com/MatrixAI/nixpkgs-matrix/pull/4#issuecomment-2292631189, I will rename it to `hercules-ci-gitignore` to make the separation more obvious.